### PR TITLE
doc(source): add sqlserver-cdc

### PR DESCRIPTION
**Pull request by KeXiangWang**
## Description

Add doc for `sqlserver-cdc` source, which is a paid-only feature.

## Related code PR

https://github.com/risingwavelabs/risingwave/pull/17429

## Checklist

- [ ] I have checked the doc site preview, and the updated parts look good.
- [ ] I have acquired the approval from the owner (and optionally the reviewers) of the code PR and at least one tech writer (`emile-00`, `hengm3467`, & `WanYixian`).

### CODE REVIEW

File: docs/guides/ingest-from-sqlserver-cdc.md [hunk 1]
```diff
@@ -0,0 +1,329 @@
...
+|NUMERIC[(M[,D])], DECIMAL[(M[,D])] |`numeric`, `numeric` supports values with a precision of up to 28 digits, and any values beyond this precision will be treated as `NULL`. |
+|CHAR[(M)], VARCHAR[(M)], TEXT, NCHAR[(M)], NVARCHAR[(M)], NTEXT |CHARACTER VARYING |
+|BINARY[(M)], NBINARY[(M)] |BYTEA |
+|DATE |DATE |
+|TIME |TIME |
+|SMALLDATETIME, DATETIME, DATETIME2| TIME WITHOUT TIME ZONE (assume UTC time zone) |
+|DATETIMEOFFSET |TIMESTAMP WITH TIME ZONE|
+|UUID |CHARACTER VARYING, uppercase |
+|XML |CHARACTER VARYING |
+|Other Types |No support |
```
Comment (StrikeW) [inline docs/guides/ingest-from-sqlserver-cdc.md]: may elaborate "Other Types" in the doc.

**Review by WanYixian — APPROVED** [review]
LGTM

File: docs/guides/ingest-from-sqlserver-cdc.md (hunk 1 above)
Comment (KeXiangWang) [reply docs/guides/ingest-from-sqlserver-cdc.md]: I'll add all the rest of the types here later.

**Comment by lmatz** [comment]
a "preview" feature

**Comment by KeXiangWang** [comment]
> a "preview" feature

Anything we should add to mark it as `preview`?

**Comment by lmatz** [comment]
> > a "preview" feature
> 
> Anything we should add to mark it as `preview`?

cc: @WanYixian @ShanlanLi

**Comment by ShanlanLi** [comment]
> > a "preview" feature
> 
> Anything we should add to mark it as `preview`?



> > a "preview" feature
> 
> Anything we should add to mark it as `preview`?

Yes, we are going to [add notes to features in the experimental phase](https://github.com/risingwavelabs/risingwave-docs/issues/2469), so as to remind users these features are not 100% stable. And a dedicated page gathering all such features is being drafted for better note management. So please add a note to such features and add the feature to the dedicated page in the future when you feel necessary.

For this pr, let me add the note and will add it to the page when [the page pr](https://github.com/risingwavelabs/risingwave-docs/pull/2488) is merged.

File: docs/guides/ingest-from-sqlserver-cdc.md (hunk 1 above)
Comment (KeXiangWang) [reply docs/guides/ingest-from-sqlserver-cdc.md]: Added

**Review by StrikeW — APPROVED** [review]

**Comment by KeXiangWang** [comment]
> For this pr, let me add the note and will add it to the page when https://github.com/risingwavelabs/risingwave-docs/pull/2488 is merged.

Please! Except that, the PR is ready to merge.

**Pull request merged**